wgsl/index.bs

```diff
@@ -5125,19 +5125,14 @@ single texel is returned.
 Reads a single texel from a texture without sampling or filtering.
 
 ```rust
-textureLoad(t : texture_1d<T>, coords : i32) -> vec4<T>
-textureLoad(t : texture_1d_array<T>, coords : i32, array_index : i32) -> vec4<T>
-textureLoad(t : texture_2d<T>, coords : vec2<i32>) -> vec4<T>
+textureLoad(t : texture_1d<T>, coords : i32, level : i32) -> vec4<T>
+textureLoad(t : texture_1d_array<T>, coords : i32, array_index : i32, level : i32) -> vec4<T>
```

Comment:
MSL:

> The following member functions can perform sampler-less reads from a 1D texture array. Since
mipmaps are not supported for 1D textures, lod must be 0.